base/src/SoundRecord.cpp

```diff
@@ -0,0 +1,146 @@
+#include "SoundRecord.h"
+#include "Module.h"
+#include "Logger.h"
+#include "SFML/System.hpp"
+#include "SFML/Graphics.hpp"
```

Comment:
Interesting to see that Graphics.hpp is required, can you confirm what happens if you remove it ?